debezium-core/src/main/java/io/debezium/pipeline/txmetadata/TransactionMonitor.java

```diff
@@ -133,6 +135,15 @@ public void transactionComittedEvent(OffsetContext offset) throws InterruptedExc
         offset.getTransactionContext().endTransaction();
     }
 
+    public void transactionComittedEvent(OffsetContext offset, long transactionEndTime) throws InterruptedException {
```

Comment:
This and the other new method are almost same as methods without `transactionEndTime`. Could you please refactor the code so the methods ae not copied but one calls another?